microservices/product-search-export/README.md

```diff
@@ -18,8 +18,7 @@ This leanest Symfony version is highly optimized and it is suitable for this typ
 The microservice is responsible for feeding the Elasticsearch by product data so [product search microservice](https://github.com/shopsys/microservice-product-search) can work properly.
 
 ## Installation
-Basically, in the Docker container of the microservice, all the dependencies have to be installed using `composer install` and the server has to be started by `php bin/console server:run *:8000`.
-All this is automatically done when the container starts.
+Microservice is installed during `docker-compose up -d` in your `project-base`, during build of image all dependencies gets installed and nginx with php-fpm si configured to allow traffic into microservice.
```

Comment:
The installation of a microservice should be independent of the rest of the framework imo, I would just mention starting the container, without mentioning project-base or docker-compose.

Also, later in this section you left the mention about [the native installation of microservices](https://github.com/shopsys/shopsys/blob/ph-mc-microservices-webserver/docs/installation/native-installation.md#22-set-up-the-microservice-for-product-search) which still recommends using the symfony webserver - but you have uninstalled it in this PR.

Reply:
I added commit that fixes this. Can you please look at it? Last commit will be squashed with README.md commit and i will use commit message "Updated documentation to match new state of microservices"